docs/_templates/class.rst

```diff
@@ -1,20 +1,61 @@
+.. |img| replace:: ``img`` (:obj:`numpy.array`): a numpy array (height, width, channels)
+   representation of the image, in BGR format
+
+.. |bboxes| replace:: ``bboxes`` (:obj:`numpy.array`): a numpy array (N,4)
+   containing bounding box information of detected objects. N corresponds to the number of objects detected,
+   and each bounding box is represented as (x1,y1,x2,y2),
+   (x,y) coordinates of the top-left corner and bottom right corner of the bounding box respectively.
+
+.. |bbox_labels| replace:: ``bbox_labels`` (:obj:`numpy.array`): a numpy array (N) strings,
+   representing the labels of detected objects. The order corresponds to `bboxes` and `bbox_scores`.
+
+.. |bbox_scores| replace:: ``bbox_scores`` (:obj:`numpy.array`): a numpy array (N). [0,1]
+   of the confidence scores for detected objects. The order corresponds to `bboxes` and `bbox_labels`.
+
+.. |keypoints| replace:: ``keypoints`` (:obj:`numpy.array`): a numpy array (N, keypoints, 2) containing the
```

Comment:
1. K instead of keypoints?
2. Should it be explained that 2 is because there's x and y? Will be clearer to users. Since we're not including an example for what it looks like in the previous node_data_glossary, we should make this as clear as possible. Same goes for some of the other less obvious ones.
3. Spelling of 'individual'
4.  Same for keypoint_scores below

Reply:
updated for both keypoints and keypont scores